compiler/pipes/gen-tree-postprocess.cpp

```diff
@@ -252,5 +252,71 @@ VertexPtr GenTreePostprocessPass::on_exit_vertex(VertexPtr root) {
     }
   }
 
+  if (auto array = root.try_as<op_array>()) {
+    return convert_array_with_spread_operators(array);
```

Comment:
You launch this function (and a loop inside it) for every array in code. While it's acceptable, it's not good, as it leads to excessive computations even when there is no spread operator (99.99% cases).
To avoid this, you can add a field `was_spread_operator_` to this pass, and then use `on_exit_vertex` checking for `op_array` both with this flag set. Be careful with nesting.